src/modules/test/recipes/HostNameTests.json

```diff
@@ -0,0 +1,140 @@
+[
+  {
+    "ComponentName": "HostName",
+    "ObjectName": "name",
+    "Desired": false,
+    "ExpectedResult": 0
+  },
+  {
+    "ComponentName": "HostName",
+    "ObjectName": "hosts",
+    "Desired": false,
+    "ExpectedResult": 0
+  },
+  {
+    "ComponentName": "HostName",
+    "ObjectName": "invalidObjectName",
+    "Desired": false,
+    "ExpectedResult": 22
+  },
+  {
+    "ComponentName": "invalidComponentName",
+    "ObjectName": "name",
+    "Desired": false,
+    "ExpectedResult": 22
+  },
+  {
+    "ComponentName": "HostName",
+    "ObjectName": "",
+    "Desired": false,
+    "ExpectedResult": 22
+  },
+  {
+    "ComponentName": "",
+    "ObjectName": "name",
+    "Desired": false,
+    "ExpectedResult": 22
+  },
+  {
+    "ComponentName": "",
+    "ObjectName": "",
+    "Desired": false,
+    "ExpectedResult": 22
+  },
+  {
+    "ComponentName": "HostName",
+    "ObjectName": "name",
+    "Desired": true,
+    "Payload": "{}",
+    "ExpectedResult": 22
+  },
+  {
+    "ComponentName": "CommandRunner",
+    "ObjectName": "commandArguments",
+    "Desired": true,
+    "Payload": "{\"commandId\": \"saveHostNameInTempFile\", \"arguments\": \"echo $(hostname) > HostNameTestsTemp.txt\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 3}",
```

Comment:
>"Payload": "{\"commandId\": \"saveHostNameInTempFile\", \"arguments\": \"echo $(hostname) > HostNameTestsTemp.txt\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 3}", [](http://example.com/codeflow?start=4&length=181)

There are no space separators in actual payloads. Suggest removing, here and elsewhere.

Reply:
Agree, addressed this in the latest iteration.